src/reference/asciidoc/whats-new.adoc

```diff
@@ -260,3 +260,7 @@ See <<xmpp-extensions>> for more information.
 The `WireTap` `ChannelInterceptor` now can accept a `channelName` which is resolved to the target `MessageChannel`
 later, during the first active interceptor operation.
 See <<channel-wiretap>> for more information.
+
+==== ChannelMessageStoreQueryProvider
+
+The `ChannelMessageStoreQueryProvider` now supports H2 database. See <<jdbc-message-store-channels>> for more information.
```

Comment:
Minor too, but need to know as well: AsciiDoc [recommendations](http://asciidoctor.org/docs/asciidoc-recommended-practices/) to start a new Sentence from a new line.

H-m, @garyrussell , we still wrap our sentences on the 120 length. Should we break this rule for our docs? :smile: